diff --git a/vite.config.ts b/vite.config.ts
index 899b057..f793764 100644
--- a/vite.config.ts
+++ b/vite.config.ts
@@ -96,6 +96,7 @@ async function ensureFavicons() {
 }

 function parseChangelog() {
+    return [];
     return execSync('git log --grep "^Changelog" -10')
         .toString()
         .split(/^commit /m)
@@ -160,6 +161,7 @@ async function getContributorsForRepo(
     repoName: string,
     githubToken?: string,
 ): Promise<Contributor[]> {
+    return [];
     const contributorsData = await fetch(
         `https://api.github.com/repos/LiveSplit/${repoName}/contributors`,
         {
@@ -300,11 +302,8 @@ export default defineConfig(async ({ mode }) => {
         console.warn("Failed to load GitHub data, skipping:", e);
     }

-    const commitHash = execSync("git rev-parse --short HEAD").toString();
-    const date = new Date()
-        .toISOString()
-        .replace("T", " ")
-        .replace(/\..+/, " UTC");
+    const commitHash = "@shortRev@"
+    const date = "@date@";

     return {
         plugins: [
